kitsune/search/v2/signals/tests/test_forums.py

```diff
@@ -0,0 +1,40 @@
+from kitsune.search.v2.tests import Elastic7TestCase
+from kitsune.forums.tests import PostFactory
+from elasticsearch7.exceptions import NotFoundError
+
+from kitsune.search.v2.documents import ForumDocument
+
+
+class ForumDocumentSignalsTests(Elastic7TestCase):
+    def setUp(self):
+        self.post = PostFactory()
+        self.post_id = self.post.id
```

Comment:
we can skip this assignment

Reply:
It's needed when testing getting a deleted document raises the correct error, otherwise we'll attempt to fetch with id `self.post.id` which will error out itself (`self.post being None`).